dotnet.el

```diff
@@ -56,7 +56,7 @@
   "Build a .NET project."
   (interactive)
   (let* ((target (dotnet-select-project-or-solution))
-         (command "dotnet build -v n \"%s\""))
+         (command "dotnet build -v n /p:GenerateFullPaths=true \"%s\""))
```

Comment:
NOTE: this is necesssary because the paths in errors are relative to the project rather than the solution file.